distributed/tests/test_scheduler.py

```diff
@@ -2201,3 +2201,16 @@ async def test_configurable_events_log_length(c, s, a, b):
     assert s.events["test"][0][1] == "dummy message 2"
     assert s.events["test"][1][1] == "dummy message 3"
     assert s.events["test"][2][1] == "dummy message 4"
+
+
+@gen_cluster()
+async def test_get_worker_monitor_info(s, a, b):
+    res = await s.get_worker_monitor_info()
+    ms = ("cpu", "time", "read_bytes", "write_bytes", "num_fds")
+    if WINDOWS:
+        ms = ("cpu", "time", "read_bytes", "write_bytes")
+    for w in (a, b):
+        for m in ms:
+            assert res["range_query"][w.address][m] is not None
```

Comment:
This check feels a little vague, I would like to do an exact check on the length of this / the value of `monitor.count` but not really sure if there's a reliable way to make sure these are consistent across tests.